### PR TITLE
Release notes for 1.50 Beta 1

### PR DESCRIPTION
**Pull request by shiftkey**
Aiming to ship whatever is on `master` to beta users tomorrow (~10am PST), and here's the current changelog.

### CODE REVIEW

**Comment by shiftkey** [comment]
Bumped to indicate this is ready for a review and added the estimated time to cut the release

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
   "releases": {
+    "1.5.0-beta1": [
+      "[New] Repository switcher has a convenient \"Add\" button to add other repositories - #5878",
+      "[New] macOS tray icon now supports drag-and-drop to add local repositories - #5048",
+      "[Added] Keyboard shortcut for \"Rename Branch\" menu item - #5964. Thanks @agisilaos!",
+      "[Added] Commit file list now has \"Copy File Path\" context menu action - #2944. Thanks @Amabel!",
+      "[Fixed] Launching Desktop from browser always asks to clone repository - #5913",
+      "[Fixed] Publish dialog displayed on push when repository is already published - #5936",
+      "[Improved] Sanitize repository name before publishing to GitHub - #3090. Thanks @Daniel-McCarthy!",
+      "[Improved] Avoid optional locks when checking status of repository - #5952",
```
Comment (outofambit) [inline changelog.json]: i'm not sure this should be a release note. or we maybe we just need to rephrase it to highlight the user-facing part.

Comment (outofambit) [reply changelog.json]:
```suggestion
      "[Improved] Getting a repository's status only blocks other operations when absolutely necessary - #5952",
```

**Comment by shiftkey** [comment]
:apple:
